datajoint/__init__.py

```diff
@@ -81,7 +81,7 @@ def create_virtual_module(modulename, dbname):
     :return: the python module
     """
     mod = ModuleType(modulename)
-    s = schema(dbname, mod.__dict__)
-    s.spawn_missing_classes()
+    s = schema(dbname, create_tables=False)
```

Comment:
Good point and yes, I agree.  However, let's make it configurable by adding a `create_if_missing` argument to `create_virtual_module`.   Let's do it with the next feature release when we rename `create_tables` throughout.

Reply:
I think `create_if_missing` should default to `False` for in `create_virtual_module` to avoid excessive schema creation due to simple mispellings.

Reply:
Ok I agree that making it configurable would be best. But for now, shall we just remove the `create_tables`? I want the default to be `create_tables=True` (that is in future, `create_if_missing=True` to be the default).

Reply:
I'm now thinking that there should be separate flags for `create_schema` and `create_tables`. I can see your point of wanting to avoid creating a schema if it doesn't already exist when using `create_virtual_module`, but I would still want to use it to create additional tables in the schema. So I think default should be `create_schema=False`, `create_tables=True`.

Reply:
I am okay with adding this change to this PR since it does not affect existing functionality.